src/main/java/ui/components/NavigableListView.java

```diff
@@ -45,7 +46,9 @@
     // (when nothing is selected, both should be no-ops)
     private boolean saveSelectionCalled = false;
 
-    private IntConsumer onItemSelected = i -> {};
+    public static final boolean SHOULD_LOAD_BVIEW_ISSUE = true;
```

Comment:
Yup, we want a selection event to occur on a right click. The issue should be selected on a right click but the bView should not open. The relevant context menu opens up for the issue on a right click.